src/views/LoginForm/LoginForm.jsx

```diff
@@ -0,0 +1,171 @@
+import React, { useState } from "react";
+import PropType from "prop-types";
+import { loginFormInputs } from "./loginFormConstants";
+import { login } from "../../utils/requests/UserRequests";
+import {
+  Alert,
+  Tooltip,
+  Button,
+  Form,
+  FormGroup,
+  FormInput
+} from "shards-react";
+import "./styles.scss";
+
+export const LoginForm = ({ handleButtonClick, onChange, hasErrors }) => {
+  return (
+    <div className="login__form">
+      <Form>
+        <div className="form-inputs">
+          {loginFormInputs.map((inputs, key) => (
+            <FormGroup key={key} onChange={onChange}>
+              <FormInput
+                className="mb-2"
+                type={inputs.type}
+                name={inputs.name}
+                id={inputs.name}
+                placeholder={inputs.placeholder}
+              />
+            </FormGroup>
+          ))}
+        </div>
+        <FormGroup>
+          <Button
+            size="lg"
+            pill
+            theme="dark"
+            onClick={event => handleButtonClick(event)}
+            name="submit"
+          >
+            {"Login"}
```

Comment:
useless curly braces.

```suggestion
            Login
```